cli/fathom_web/test/test_test.py

```diff
@@ -0,0 +1,59 @@
+from click import BadParameter
+import pytest
+
+from ..commands.test import decode_weights
+
+
+def test_valid_json_input():
```

Comment:
I could use a docstring on this one; I wasn't immediately sure what it did from its name. I just got it on my 3rd reading.